src/notte/browser/pool/base.py

```diff
@@ -29,6 +29,7 @@ class BrowserResource:
     browser_id: str
     context_id: str
     headless: bool
+    port: int | None = None
```

Comment:
TODO: still need to add code to fill this value in the local pool when the resource is created